README.md

```diff
@@ -106,7 +106,7 @@ jobs:
       - slack/status:
           mentions: "USERID1,USERID2" # Optional: Enter the Slack IDs of any user or group (sub_team) to be mentioned
           fail_only: true # Optional: if set to `true` then only failure messages will occur.
-          webhook: "webhook" # Optional: Enter a specific webhook here or the default will use $SLACK_WEBHOOK
+          webhook: ${SLACK_WEBHOOK} # Please use the CircleCI UI to set the slack webhook under the SLACK_WEBHOOK environment variable 
```

Comment:
You would not use ${} here. This would be in the source of the command. This would just be the string of the webhook.

Reply:
Same for similar references else where in these changes.